src/MCPClient/lib/settings/common.py

```diff
@@ -16,20 +16,20 @@
 # along with Archivematica.  If not, see <http://www.gnu.org/licenses/>.
 
 import os, sys, ConfigParser
-import django_mysqlpool
 
 # Get DB settings from main configuration file
 config = ConfigParser.SafeConfigParser()
 config.read('/etc/archivematica/archivematicaCommon/dbsettings')
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django_mysqlpool.backends.mysqlpool',
+        'ENGINE': 'django.db.backends.mysql',
         'NAME': config.get('client', 'database'),          # Or path to database file if using sqlite3.
         'USER': config.get('client', 'user'),              # Not used with sqlite3.
         'PASSWORD': config.get('client', 'password'),      # Not used with sqlite3.
         'HOST': config.get('client', 'host'),              # Set to empty string for localhost. Not used with sqlite3.
         'PORT': '',                                        # Set to empty string for default. Not used with sqlite3.
+        'CONN_MAX_AGE': 500,
```

Comment:
Something to remember is that Django achieves thread-safety by not sharing connections between threads.

https://docs.djangoproject.com/en/dev/ref/databases/#caveats
> Since each thread maintains its own connection, your database must support at least as many simultaneous connections as you have worker threads.

With this caveat in mind, my two concerns related to scalability issues are:

- We are deploying gunicorn using the `sync` [worker class](http://docs.gunicorn.org/en/stable/settings.html#worker-class) (synchronous workers). Under this concurrency model, persistent connections created within a worker will be successfully reused on new requests processed by the same worker. However, if we ever switch to greenlets (`eventlet` or `gevent` worker class) - which we should seriously consider for performance and reliability reasons, persistent connections will become an issue because Django's thread-safety model will force these green threads to make new connections on each request. These connections are not being closed by the client and will accumulate.

- In MCPServer, we create a significant amount of POSIX threads. By adding a pool we allowed these threads to share and reuse existing connections and keep the total number of connections to the database as low as the size of the pool. To make sure threads return connections to the pool as soon as possible we used the `@auto_close_db` decorator. It also helped to reduce thread lock contention. Not having a pool I believe puts us back to a concurrency model where connections are not shared and created on a per-thread basis.

If we stop using our pool, we should at least make sure that MCPServer's threaded functions close the database connections as soon as they return. Simplest solution would be to implement our own `@auto_close_db` decorator. The fundamental reason why the MCPServer needs extra care is because Django has its own mechanism to close these connections when a web request starts and/or finishes - this mechanism just never happens when you are using Django's ORM outside the web context.